packages/devtools/lib/src/ui/ui_utils.dart

```diff
@@ -29,19 +33,86 @@ String userAppType = ''; // dimension1
 String userBuildType = ''; // dimension2
 String userPlatformType = ''; // dimension3
 
-void computeApplicationState(
-  bool isAnyFlutterApp,
-  bool isFlutter,
-  bool isWebApp,
-  bool isProfile,
-  bool isAndroid,
-) {
+String devtoolsPlatformType = ''; // dimension4 MacIntel/Linux/Windows/Android_n
+String devtoolsChrome = ''; // dimension5 Chrome/n.n.n  or Crios/n.n.n
+const String devtoolsVersion = devtools.version; //dimension6 n.n.n
+
+bool get isDimensionsComputed =>
+    (userAppType.length +
```

Comment:
nit:
```dart
userApType.isNotEmpty || userBuildType.isNotEmpty || ...;
```
is more idiomatic Dart.

Reply:
These lines were eliminated - I'll upload the new files.